JMS/target/maven-archiver/pom.properties

```diff
@@ -0,0 +1,4 @@
+#Created by Apache Maven 3.8.6
```

Comment:
The target directory and its contents should not be part of this repository. Please remove from the pull request.